fail2ban/tests/utils.py

```diff
@@ -107,6 +107,7 @@ def addTest(self, suite):
 	tests.addTest(unittest.makeSuite(failmanagertestcase.AddFailure))
 	# BanManager
 	tests.addTest(unittest.makeSuite(banmanagertestcase.AddFailure))
+	tests.addTest(unittest.makeSuite(banmanagertestcase.StatusExtendedCymruInfo))
```

Comment:
Agreed. It is installed for Travis tests, but added import test around it. a80ff26e151c295bb5cc1661fa5402c0115a8e97